demisto_sdk/commands/content_graph/tests/test_data/layoutrule.json

```diff
@@ -0,0 +1,18 @@
+{
```

Comment:
why do we need this file? isn't the default data of TestSuite.LayoutRule enough?

Reply:
it is used in the ```test_layout_rule_parser``` function to create rule using the TestSuit for convenience.
Remove it and insert the json in the code itself?